src/components/CartIcon/index.js

```diff
@@ -0,0 +1,15 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
Acho que você esqueceu os imports aqui, certo?
```suggestion
import React from 'react';
```